alpha/veneer/semver/semver.go

```diff
@@ -18,7 +18,7 @@ import (
 
 // data passed into this module externally
 type Veneer struct {
-	Ref string
+	Ref io.Reader
 	Reg image.Registry
```

Comment:
Totally non-blocking, but this was crap when I wrote it, and I meant to get back to it, but ... I lost track.  
If I had a wish-list, I would change this to something like 
```
type Veneer struct {
   Data     io.Reader
   Registry image.Registry
}
```
... and usage accordingly.  
I mention it because in here there's already a breaking API change, so it wouldn't introduce one. ;)